app/views/pages/cleaners_responsibility.html.haml

```diff
@@ -0,0 +1 @@
+%h1 Please raise this issue with the cleaner
```

Comment:
NB: I've added this page to the [static content card](https://trello.com/c/OyIiRYa3/146-content-for-static-pages) on Trello.